spark/v3.1/spark-extensions/src/main/antlr/org.apache.spark.sql.catalyst.parser.extensions/IcebergSqlExtensions.g4

```diff
@@ -162,6 +163,10 @@ booleanValue
     : TRUE | FALSE
     ;
 
+stringArray
+    : ARRAY '(' constant (',' constant)* ')'
```

Comment:
I realize this is probably just copying the map rule, but this is an array of any constant, not just strings.